tests/ci-helper.test.ts

```diff
@@ -304,9 +311,10 @@ testQ("handle comment allow fail invalid user", async () => {
     };
 
     ci.setGHGetPRComment(comment);
+    ci.letGHGetGitHubUserInfoThrow("SOME ERROR");
 
-    await expect(ci.handleComment("gitgitgadget", 433865360)).rejects.toThrow(/is not a valid GitHub username/);
-    expect(ci.addPRCommentCalls[0][1]).toMatch(/is not a valid GitHub username/);
+    await expect(ci.handleComment("gitgitgadget", 433865360)).rejects.toThrow(/SOME ERROR/);
```

Comment:
The purpose of the test is to verify the bad user is reported by CIHelper.  Verifying the network access was bypassed seems like a different test.  In the CI test, github throws a bad request type error that CIHelper converts to the bad user with the GitHubGlue error message as part of the error.  Running locally, the test actually gets  a not authenticated error thrown by GitHubGlue (so not quite the test I thought it was).  Setting 
`getConfig().repo.owner = "gooduser";` fixes the auth error if the config is properly set up.  I understand what the update is proofing but don't think it is part of the test - the checks do not need to be changed.

Reply:
Wait, you mean to say that the patch avoiding network access to GitHub REST should be dropped?

Reply:
Running locally, there is no network access due to no authentication (required to ask about users).  Running on workflows or pipelines, there is network access but is that a problem?  I'll let you decide on that.

I think, either way, the tests should still be checking for `is not a valid GitHub username` and not `SOME ERROR` since that is the purpose of the test.  CIHelper is setting the not valid message.  Sorry I was not clear.